scripts/create-k3s-cluster-with-SSL-passthrough.sh

```diff
@@ -99,4 +107,4 @@ if [[ "$wait" == "true" ]] ; then
     --for=condition=ready pod \
     --selector=app.kubernetes.io/component=controller \
     --timeout=90s
-fi
+fi
```

Comment:
It was not necessary to delete the linebreak at the end of the file.

Reply:
Okay.